lib/pdex/nppes_data_loader.rb

```diff
@@ -15,13 +15,20 @@ class NPPESDataLoader
 
     class << self
     include ShortName
-    def load
+    def load(mode)
         load_managing_organizations
         load_networks
         load_organizations
```

Comment:
Are those tiny enough so we don't have to regenerate them via optional mode? Or are they required for both providers and pharmacy modes?

Reply:
They're tiny